app-backend/batch/src/main/scala/com/azavea/rf/batch/landsat8/ImportLandsat8.scala

```diff
@@ -0,0 +1,281 @@
+package com.azavea.rf.batch.landsat8
+
+import com.azavea.rf.batch.Job
+import com.azavea.rf.batch.util._
+import com.azavea.rf.database.{Database => DB}
+import com.azavea.rf.database.tables._
+import com.azavea.rf.datamodel._
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import io.circe._
+import io.circe.syntax._
+import geotrellis.proj4.CRS
+import geotrellis.slick.Projected
+import geotrellis.vector._
+import jp.ne.opt.chronoscala.Imports._
+import java.time.{LocalDate, ZoneOffset}
+import java.util.UUID
+
+import scala.io.Source
+import scala.collection.mutable.ListBuffer
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, Future}
+import scala.util.{Failure, Success}
+import scala.util.control.Breaks._
+
+case class ImportLandsat8(startDate: LocalDate = LocalDate.now(ZoneOffset.UTC), threshold: Int = 10)(implicit val database: DB) extends Job {
+  val name = "import_landsat8"
+
+  implicit val system       = ActorSystem(s"$name-system")
+  implicit val materializer = ActorMaterializer()
+
+  @SuppressWarnings(Array("OptionGet"))
+  protected def getCsv: ListBuffer[Map[String, String]] = {
+    val source = Source.fromURL(landsat8Config.usgsLandsatUrl, "UTF-8")
```

Comment:
If its not broke don't fix it, but in general `IOUtils` are safer and more performant way to read an input stream than scala `Source`.